api.py

```diff
@@ -77,7 +240,11 @@ def get(self, args):
 		self.error(404)
 
 app = webapp2.WSGIApplication([
-	('/api/members', MemberListAPI),
-	('/api/member', MemberAPI),
+	('/api/members/?', MemberListAPI),
+	('/api/rank/([a-zA-Z0-9]+)', RankAPI),
+	('/api/missions/?', MissionListAPI),
+	('/api/mission/([a-z0-9]+)', MissionAPI),
+	('/api/bridgecrews/?', BridgeCrewListAPI),
+	('/api/bridgecrew/([a-z0-9]+)?', BridgeCrewAPI),
```

Comment:
Either `/rank` should not accept `A-Z`, or `/mission` and `/bridgecrew` should accept.  I am inclined to say the latter so the error comes from it being an invalid ID requested rather than appearing to be an invalid API endpoint.

But then might it be better to just make those each `(.+)` (or `([^/]+)`) and have the functions handle the error states?